client/src/components/reader/TocDrawer.js

```diff
@@ -18,13 +19,16 @@ export default class TocDrawer extends Component {
       'drawer-visible': this.props.visible
```

Comment:
Not part of this commit, but something we could fix. Classnames takes as many arguments as you need. So instead of this:
```
classNames({
  'toc-drawer': true,
  'drawer-hidden': !this.props.visible,
  'drawer-visible': this.props.visible
});
```
we could do:
```
classNames("toc-drawer", {
  'drawer-hidden': !this.props.visible,
  'drawer-visible': this.props.visible
});
```